### DISCUSSION TITLE
Sound

### DISCUSSION THREAD

**Discussion by Mr-Tech-13** [jef/streetmerchant#2638] [Help] [answered]
I get this error on windows 10 while trying to play the sound.
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
INFO: Could not find files for the given pattern(s).
[1:56:05 PM] warn :: ✖ couldn't find sound player

The sound is a mp3 in my downloads folder

**Comment by WeDoSys** [comment, marked as answer]
https://github.com/jef/streetmerchant/wiki/Installation:-Windows#audio-notifications

**Reply by Mr-Tech-13** [reply]
Thank you
